Physics Demo

afourneaux opened the pull request:
Created a new scene (accessible from main menu) to demonstrate the collision resolution system.

Additional changes:
Smoothed the collision resolution system
Deleted unused main menu scene and MainMenuController.cs

afourneaux committed:
Physics Demo

WendyFu004 commented:
That was cool, look nice from my side. Only one suggestion is could you change to or add one of our racer model to the scene so it will look like in our game?

Or add one bird AI area with birds, so we could demo our group AI and more tech in this scene and we don't need a spectator mode in real time game.

afourneaux committed:
Added rocket model to PhysicsDemo

afourneaux commented:
Added rocket model to physics demo. Also fixed a bug where many player models had two collision root scripts.

As to demonstrating Bird AI, while I agree it would be good to have an isolated demo of it, adding it to a Collisions demo would be the wrong place. Bird AI demo would require a large scene with AI rockets circling around the Bird area, entering and exiting the area trigger. Considering this is the final day before submission, I'm not sure it would be wise to throw that together now

WendyFu004 commented:
You're right. We don't do trouble anymore in the last day
